### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by dazehinn**
* created a Request spec file for the posts controller
* created a Request spec file for the users controller

### CODE REVIEW

File: spec/posts_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
```
Comment (OLIPLICHE) [inline spec/posts_controller_spec.rb]: - Hi, Can you create a directory called requests as it is said in the requirement(Create a Request spec file for all your controllers.)? and move your controllers test inside?
**Hint**:  `users_controller_spec.rb` and `posts_controller_spec.rb` will be in `spec/requests/` directory

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @dazehinn, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- RSpec gem is used for testing 👍 
- index and show actions for user and post controllers were tested 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @dazehinn 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Correct file structure
✔️ All tests are of type request

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @dazehinn, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that's meant for different milestone. You're to submit the PR meant for `add forms` milestone not for the `Controllers specs` milestone. Kindly do just that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by dazehinn** [comment]
Hello @Qoosim. It was actually an error while trying to submit the changes requested by the previous reviewer. Sorry for wasting your time.

**Comment by Qoosim** [comment]
> Hello @Qoosim. It was actually an error while trying to submit the changes requested by the previous reviewer. Sorry for wasting your time.

@dazehinn, don't worry. Thanks for appreciating my time.

**Pull request merged**